stdlib/sys.pyi

```diff
@@ -222,7 +223,7 @@ if sys.version_info >= (3, 11):
     def exception() -> BaseException | None: ...
 
 # sys.exit() accepts an optional argument of anything printable
```

Comment:
This comment is now out of date. Can you delete it, and add a comment above the definition of sys._ExitCode, summarising the rationale laid out in https://github.com/python/typeshed/issues/8513#issue-1333671093 for why we're going with `str | int | None` here? (You could maybe just link to that comment in the comment.)

Reply:
Added a link to the comment as I think to do the full rationale could end up being rather verbose

Reply:
Looks good :)